internal/backends/winit/renderer/femtovg/images.rs

```diff
@@ -101,6 +101,11 @@ impl Texture {
                     // pre-multipled alpha. It's possible that this is not generally applicable, but it
                     // is the case for SVGs.
                     let image_flags = if html_image.dom_element.current_src().ends_with(".svg") {
```

Comment:
```suggestion
                    let image_flags = if html_image.is_svg() {
```

(that was the whole point of moving it to a function)

Reply:
🤦‍♂️ thanks!